tests/bluetooth/bsim_bt/bsim_test_multiple/prj.conf

```diff
@@ -43,3 +43,4 @@ CONFIG_BT_CTLR_RX_BUFFERS=6
 # accuracy, current value here is sufficient for 500ppm at 1 second interval.
 CONFIG_BT_CTLR_ADVANCED_FEATURES=y
 CONFIG_BT_CTLR_SCHED_ADVANCED_CENTRAL_CONN_SPACING=1000
+CONFIG_BT_CTLR_SCHED_ADVANCED=y
```

Comment:
Why turn of in Kconfig file and enable `CONFIG_BT_CTLR_SCHED_ADVANCED` here?

This babblesim test is reuse of https://github.com/zephyrproject-rtos/zephyr/tree/main/samples/bluetooth/central_multilink  and https://github.com/zephyrproject-rtos/zephyr/tree/main/samples/bluetooth/peripheral_identity, have you tested them on target?

Reply:
Fixed